components/scroller/scroller.vue

```diff
@@ -0,0 +1,642 @@
+<template>
```

Comment:
I'm guessing this is all forked library code? We cannot dump all this directly in the component, we should maintain a separate forked repo with the library code, make our modifications to it and publish it separately. This will allow us to merge in any updates and make it much easier to see exactly what we changed from the original code. This component is much too complex, it should be using the external library rather than implementing it.